pos_takeout/models/pos_order.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 KMEE INFORMATICA LTDA (http://www.kmee.com.br).
+# @author: Felipe Zago Rodrigues <felipe.zago@kmee.com.br>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    eat_here = fields.Boolean(string="Is the customer eating here", default=True)
```

Comment:
To be consistent with the module name and the rest of the terms, I'd invert this field like this:

```suggestion
    takeout = fields.Boolean(help="Is this a takeout order?")
```